qa/common/gen_qa_model_repository

```diff
@@ -512,7 +512,7 @@ chmod -R 777 $FORMATDESTDIR
 python3 $SRCDIR/gen_qa_trt_data_dependent_shape.py --models_dir=$DATADEPENDENTDIR
 chmod -R 777 $DATADEPENDENTDIR
 # Make shared library for custom Hardmax plugin.
-(git clone -b release/10.0 https://github.com/NVIDIA/TensorRT.git && \
+(git clone -b release/10.5 https://github.com/NVIDIA/TensorRT.git && \
```

Comment:
This value is hardcoded and may bother us on next release, I would suggest following: 
```suggestion
(git clone -b release/$(echo ${TRT_VERSION} | cut -d . -f -2) https://github.com/NVIDIA/TensorRT.git && \
```
Sure it's not a very flexible solution, but at least it will give us some time to stay away of changing it each release.